tests/unit/models/test_user.py

```diff
@@ -0,0 +1,22 @@
+FOLLOWING_USER_ID = '2'
+MOVIE_ID = '123'
+
+
+def test_adding_user_id_to_followed(user):
+    followed_users_ids = user.get_followed_users_ids()
+    assert len(followed_users_ids) == 0
+
+    user.add_to_followed_users_ids(FOLLOWING_USER_ID)
+    followed_users_ids = user.get_followed_users_ids()
+    assert len(followed_users_ids) == 1
+    assert FOLLOWING_USER_ID in followed_users_ids
+
+
+def test_adding_movie_id_to_liked(user):
+    liked_movies_ids = user.get_liked_movies_ids()
+    assert len(liked_movies_ids) == 0
+
+    user.add_to_liked_movies_ids(MOVIE_ID)
+    liked_movies_ids = user.get_liked_movies_ids()
+    assert len(liked_movies_ids) == 1
+    assert MOVIE_ID in liked_movies_ids
```

Comment:
That will probably be useful at some point, however, I suggest that we start with a test concerning only `RecommendationService` and leave the model specific API tests until we know that they are in fact required :)